### DISCUSSION TITLE
what happend for this exception

### DISCUSSION THREAD

**Discussion by laglangyue** [apache/pulsar#20019] [Q&A]
STDOUT: 2023-04-05 14:30:05,358 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140159899155200 for <Subprocess at 140159899154912 with name pulsar in state RUNNING> (stdout)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|221] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])

**Comment by laglangyue** [comment]
I notice a exception when start bellow
```log
org.apache.zookeeper.ClientCnxn - Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error)
```

**Reply by tisonkun** [reply]
This is not an exception, but tells you which mode is turned on.

**Comment by tisonkun** [comment]
The report is quite vague that seems having no valuable information.

cc @BewareMyPower is it a typical Python client issue? If not, I tend to close this one as information incompleted.

**Reply by BewareMyPower** [reply]
No. Since it uses Python 2.7, it could be a very old client version. Python client 3.x only supports Python 3.7 or higher.
